packages/babel-types/src/definitions/jsx.ts

```diff
@@ -107,6 +107,7 @@ defineType("JSXIdentifier", {
 
 defineType("JSXMemberExpression", {
   visitor: ["object", "property"],
+  aliases: ["Expression"],
```

Comment:
Is it ok to do this change in the same PR?

Reply:
Or perhaps it's better to extend `this` type for `referencesImport` with concrete `JSXMemberExpression` type 🤔